### PR TITLE
Merge changes from master branch

### PR DESCRIPTION
**Pull request by esme**
## Description

Merge changes from master branch.

## Merge Checklist

| Q                        | A 
| ------------------------ | --------------
| Adds Documentation?      |
| Any Dependency Changes?  |
| Patch: Bug Fix?          |
| Minor: New Feature?      |
| Major: Breaking Change?  |

[BRAVE Checklist](https://github.com/HubSpot/calling-extensions-sdk/blob/master/SHIP_WITH_CARE.md)

- [x] I have read the BRAVE checklist and confirmed if the following is necessary.

| Q                              | A 
| ------------------------------ | --------------
| Backwards Compatible?          |
| Rollout/Rollback Plan?         | 
| Automated test coverage?       | 
| Verified that changes work?    |
| Expect Dependencies to Fail?   |

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-v14.21.3
+v18.15.0
```
Comment (alonso-cadenas) [inline .node-version:1]: Do we already have a ticket to delete `.node-version` files from the repo, update `.gitignore`, and add `engines` prop to `package.json`?

**Review by alonso-cadenas — COMMENTED** [review]
Is there any way to preserve git history as we merge from master into this branch? Asking because the commit section doesn't show it right now.

File: .node-version (hunk 1 above)
Comment (esme) [reply .node-version:1]: Yes we have a ticket for that. This is the change I made before creating the ticket, so the merge includes it.

**Comment by esme** [comment]
> Is there any way to preserve git history as we merge from master into this branch? Asking because the commit section doesn't show it right now.

@alonso-cadenas The master branch has the [git history](https://github.com/HubSpot/calling-extensions-sdk/commits/master) and using merge here will preserve the history over rebase. I've been using rebase so far but I'm iffy about it this time as it is tricky to rebase these many changes. Eventually, we would merge `project-demo-v1` into `master` and can squash the commits for a cleaner history.

**Review by alonso-cadenas — APPROVED** [review]

**Pull request merged**